homeassistant/components/modbus/validators.py

```diff
@@ -112,6 +114,28 @@
 }
 
 
+def modbus_create_issue(
+    hass: HomeAssistant, key: str, subs: list[str], err: str
+) -> None:
+    """Create issue modbus style."""
+    async_create_issue(
+        hass,
+        DOMAIN,
+        key,
+        is_fixable=False,
+        severity=IssueSeverity.WARNING,
+        translation_key=key,
+        translation_placeholders={
+            "sub_1": subs[0],
+            "sub_2": subs[1],
+            "sub_3": subs[2],
+            "integration": DOMAIN,
+            "url": "https://www.home-assistant.io/integrations/modbus",
```

Comment:
Use the `async_create_issue` options so set the docs URL or the entity platform that has an issue.
    issue_domain: str | None = None,
    learn_more_url: str | None = None,

Reply:
Done,

I actually thought that the second parameter "domain" was the same as "issue_domain".

The "learn_more_url" is a nice tip, thanks.